from_cpython/Modules/_ctypes/_ctypes.c

```diff
@@ -2044,13 +2044,21 @@ PyCSimpleType_new(PyTypeObject *type, PyObject *args, PyObject *kwds)
             break;
         }
 
-        if (ml) {
+        // TODO: Pyston change:
```

Comment:
I think it's best if we can avoid these kinds of "soft" failures, since it'll potentially be much harder to debug why we get a `no attribute "from_param"` than a "hard" error like "Py_FatalError: unimplemented".  I tried, though, to think of some other way to get around this, and I couldn't think of anything better so let's just go with this and hope we remember that this is here :)